CaseyShimata/ACHLibrary
# Feature/loanpro 25/convert raw record array to fields

Testing: instructions:

1. Run the http://localhost:8080/ach-data-validate end point set up in postman (see the Confluence doc for all the links. https://caseyshimata.atlassian.net/wiki/spaces/LOANPRO/pages/33091/LoanProACHValiatorAndUI 
2. Verify each field for each record is filled with accurate characters, length, position etc.

Not handling ACH files with invalid lengths currently. That is in the next release.
